man/man8/zpool-create.8

```diff
@@ -179,6 +187,12 @@ Sets the given pool properties.
 See the
 .Xr zpoolprops
 manual page for a list of valid properties that can be set.
+.It Fl o Ar compatibility Ns = Ns Ar off | legacy | file Bq , Ns Ar file Ns ...
+Specifies compatibility feature sets. See the
+.Ar compatibility
+property in
+.Xr zpoolprops 8
+for information about compatibility feature sets.
```

Comment:
Once we decide on where to explain the logic behind the "standard" compatibility files, this should probably be updated to point there, as well.